gobblin-kubernetes/gobblin-service/basic-cluster/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
```

Comment:
Is `rewritten` needed as we still keep the original url here?

Reply:
I don't think so, I'll clean this up, thanks!